build.gradle

```diff
@@ -43,6 +43,8 @@ dependencies {
     testRuntime 'org.pegdown:pegdown:1.4.2' // Necessary for generating HTML reports with ScalaTest
     testCompile "org.testng:testng:6.9.9"
     testCompile "com.google.jimfs:jimfs:1.1"
+    // https://mvnrepository.com/artifact/com.google.cloud/google-cloud-nio
+    testCompile group: 'com.google.cloud', name: 'google-cloud-nio', version: '0.5.0'
```

Comment:
I would use `jimfs` instead of google-cloud for testing a different filesystem. Like that, you can avoid the issues with your GCS costs...

Reply:
done